hack/golangci-strict.yaml

```diff
@@ -11,18 +11,21 @@ issues:
   max-same-issues: 0
   # Excluding configuration per-path, per-linter, per-text and per-source
   exclude-rules:
-    # exclude ineffassing linter for generated files for conversion
+    # exclude ineffassign linter for generated files for conversion
     - path: conversion\.go
       linters:
         - ineffassign
 
 linters:
-  enable: # please keep this alphabetized
+  disable-all: false # in contrast to golangci.yaml, the default set of linters remains enabled
+  enable: # please keep this alphabetized and in sync with golangci-strict.yaml
```

Comment:
you mean `golang-ci.yaml` here on line 21 right?

Reply:
No, the file really is called `hack/golangci.yaml`.

Reply:
Oh, you mean the second line. Yes. It's the opposite of what it should be.